test/utils.js

```diff
@@ -489,3 +489,20 @@ module.exports.generateRandomCode = function (size) {
 module.exports.delay = async function (time) {
     await new Promise((r) => setTimeout(r, time * 1000));
 };
+
+module.exports.areArraysEqual = function (arr1, arr2) {
+    if (arr1.length !== arr2.length) {
+        return false;
+    }
+
+    arr1.sort();
+    arr2.sort();
```

Comment:
I think this would only work for arrays that have primitive types. So rename this function to something like `arePrimitiveTypeArraysEqual`